app/models/event.rb

```diff
@@ -3,7 +3,7 @@
 class Event < ApplicationRecord
   belongs_to :place
   belongs_to :user
-  has_and_belongs_to_many :members, class_name: 'User', before_remove: :notice_before_remove
+  has_and_belongs_to_many :members, class_name: 'User', dependent: :destroy
```

Comment:
Are you really sure about that? Do you want to remove user when you remove event?